src/test/java/com/exadel/discountwebapp/vendor/mapper/VendorMapperTest.java

```diff
@@ -0,0 +1,59 @@
+package com.exadel.discountwebapp.vendor.mapper;
+
+import com.exadel.discountwebapp.vendor.repository.VendorRepository;
+import com.exadel.discountwebapp.vendor.vo.VendorRequestVO;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+@SpringBootTest
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:sql/vendor-init.sql")
+@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts = "classpath:sql/clean-up.sql")
+public class VendorMapperTest {
+
+    @Autowired
+    private VendorMapper vendorMapper;
+
+    @Autowired
+    private VendorRepository vendorRepository;
+
+    @Test
+    public void ShouldMapEntityToVO(){
```

Comment:
method name should not start from the capital letter

Reply:
done